src/main/java/ua/hillel/moneymanager/controller/CategoryController.java

```diff
@@ -0,0 +1,54 @@
+package ua.hillel.moneymanager.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.autoconfigure.neo4j.Neo4jProperties;
+import org.springframework.data.domain.Page;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+import ua.hillel.moneymanager.dto.CategoryDto;
+import ua.hillel.moneymanager.entity.Category;
+import ua.hillel.moneymanager.service.category.CategoryServiceImpl;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@RestController
+@RequestMapping("/categories")
+@RequiredArgsConstructor
+public class CategoryController {
+
+    private final CategoryServiceImpl categoryService;
```

Comment:
Use interface `CategoryService` here and Spring will inject an implementation automatically